Automattic/jetpack
# AI Assistant: Add option to add word to Breve spelling dictionary

Fixes #39041 

## Proposed changes:

* Includes an "Add to dictionary" button next to the "Ignore" button to, well, add the word to the dictionary.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the editor and add a text with some spelling highlights
* Hover one word and click on "Add to dictionary"
* Check that the word is permanently removed from the highlights
* Reload the page
* Check that the word is still not highlighted

![2024-08-26_17-43-23](https://github.com/user-attachments/assets/04e5459a-2c96-4026-8927-d1e7ea34eb92)
